server/routerlicious/pnpm-lock.yaml

```diff
@@ -99,7 +99,8 @@ importers:
     specifiers:
       '@fluid-tools/build-cli': ^0.22.0
       '@fluidframework/build-common': ^2.0.0
-      '@fluidframework/build-tools': ^0.22.0
+      '@fluidframework/build-tools': ^0.23.0
+      '@fluidframework/common-definitions': ^1.0.0
```

Comment:
`pnpm i --frozen-lockfile` is failing with this change as there was not a corresponding change in `package.json`.

Reply:
fix PR: [https://github.com/microsoft/FluidFramework/pull/17330](https://github.com/microsoft/FluidFramework/pull/17330)